### DISCUSSION TITLE
How to customize year selector

### DISCUSSION THREAD

**Discussion by oleksandr-rybakov** [onesine/react-tailwindcss-datepicker#242] [Q&A]
I want to show user only current and previous years, how can I do it?
Maybe we can hide year selector at least?

**Comment by D-Binara** [comment]
> I want to show user only current and previous years, how can I do it? Maybe we can hide year selector at least?

 hi @oleksandr-rybakov, did you find a way to customize the year selector to show only the current and previous years?
